go/ephemeral/lib.go

```diff
@@ -179,15 +180,27 @@ func (e *EKLib) newDeviceEKNeeded(ctx context.Context, merkleRoot libkb.MerkleRo
 	s := e.G().GetDeviceEKStorage()
 	maxGeneration, err := s.MaxGeneration(ctx)
 	if err != nil {
-		return needed, err
+		switch err.(type) {
+		case *erasablekv.UnboxError:
```

Comment:
Why is this a `*` here?

Reply:
`https://github.com/keybase/client/pull/12502/files#diff-97b6f490cc3dd8c39619127caf5a83a6R24` returns a pointer. should we change?

Reply:
Yeah I'd change it to just be a normal type.

Reply:
done!